pages/workspace/workspace.js

```diff
@@ -82,4 +126,15 @@ router.get('/:workspace_id', (req, res, next) => {
     });
 });
 
+router.get('/:workspace_id/:action', (req, res, _next) => {
+    const workspace_id = req.params.workspace_id;
+    const action = req.params.action;
+
+    if (action === 'reboot') {
+        logger.info(`[workspace.js] Rebooting workspace ${workspace_id}.`);
+        controlContainer(workspace_id, 'destroy');
+        res.redirect(`/workspace/${workspace_id}`);
+    }
```

Comment:
Add an else that goes to an error page like this: https://github.com/PrairieLearn/PrairieLearn/blob/1b49e1dd1171d745f3d6b4379f3efc64c4c0a266/pages/instructorAssessmentInstances/instructorAssessmentInstances.js#L91